examples/twittermap/web/public/javascripts/mapresultcache/services.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Module to cache map result data. Common services module communicates with
+ * middleware and updates resultcache module.
+ */
+'use strict';
+angular.module('cloudberry.mapresultcache', ['cloudberry.common'])
+  .service('MapResultCache', function () {
```

Comment:
I think we are using lower cases for the service name. correct me if I am wrong.

Reply:
It's different for each service. Camel case seems to be the convention.